### PR TITLE
Wes/institution search and filter UI

### PR DESCRIPTION
**Pull request by wesrisenmay-mx**

### CODE REVIEW

File: apps/ucp-ui/src/Institutions/InstitutionFilters.tsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+          }
+          label={INSTITUTIONS_FILTER_INCLUDE_INACTIVE_INTEGRATIONS_LABEL_TEXT}
+          slotProps={slotProps}
+        />
+      </FormGroup>
+    </div>
+  );
+};
+
+export default InstitutionFilters;
```
Comment (tphalp) [inline apps/ucp-ui/src/Institutions/InstitutionFilters.tsx:183]: We previously decided to export inline, and not at the end of the file. Is that pattern different with Components?

Comment (wesrisenmay-mx) [reply apps/ucp-ui/src/Institutions/InstitutionFilters.tsx:183]: image

I just do it so I can name the the default export, and vscode will pick it up easier

Comment (tphalp) [reply apps/ucp-ui/src/Institutions/InstitutionFilters.tsx:183]: Makes sense. Thanks for explaining.

File: apps/ucp-ui/src/Institutions/InstitutionFilters.test.tsx [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+    }
+
+    const [firstAggregator] = aggregators;
+
+    await userEvent.click(screen.getByLabelText(firstAggregator.displayName));
+
+    expectedAggregatorNames.splice(
+      expectedAggregatorNames.findIndex(
+        (name) => firstAggregator.name === name,
+      ),
```
[the remark this answers is not in the document]
Comment (wesrisenmay-mx) [reply apps/ucp-ui/src/Institutions/InstitutionFilters.test.tsx]: Good call. Fixed

**Review by Craiting — APPROVED** [review]

**Pull request merged**